Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs

```diff
@@ -0,0 +1,69 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for managing the roles on a user.")]
+public class RoleManagementSlashModule : InteractionModuleBase<SocketInteractionContext> {
+    [SlashCommand("change", "Select a specific one from the role you have")]
```

Comment:
"Select a specific one from the role you have to (something)"

This sentence is incomplete and I can't tell what this command could do if I am not looking at the actual code.